src/config.c

```diff
@@ -522,6 +522,12 @@ void loadServerConfigFromString(char *config) {
                 err = "rdb-key-save-delay can't be negative";
                 goto loaderr;
             }
+        } else if (!strcasecmp(argv[0],"key-load-delay") && argc==2) {
```

Comment:
we already have rdb-key-save-delay as (undocumented) config, which i added a few months back (and was merged to unstable already).
one reason to keep these as config rather than debug is that you're able to set them at startup and they'll also affect the initial rdb/aof loading.

Reply:
@oranagran that makes sense. In retrospect maybe we should have at least named them in a way that makes it clear it's a debug configuration and not something to play with in real life.